### PR TITLE
template config changes

### PR DESCRIPTION
**Pull request by deepunbxd005**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -557,17 +557,33 @@ import environment from './environment';
         var device = getDeviceType();
         var browserSize = getBrowserSize();
         var itemsToShowOnMobile, itemWidth, itemWidthUnit;
-        if (device === MOBILE || browserSize === SMALL) {
+        
+        if(window.unbxdDeviceType === "mobile-browser" || options.unbxdDeviceType === "mobile-browser"){
             itemWidth = (rexConsoleConfigs.products && rexConsoleConfigs.products.width && rexConsoleConfigs.products.width.value) || 0;
```
Comment (skumar-lohika) [inline src/index.js:562]: please evaluate rexConsoleConfigs.products outside in a local variable and used it in all if and else condtions.

File: src/index.js [hunk 2]
```diff
@@ -557,17 +557,33 @@ import environment from './environment';
...
-            itemsToShowOnMobile = rexConsoleConfigs.products.visibleOnMobile;
+            if(rexConsoleConfigs && rexConsoleConfigs.products && rexConsoleConfigs.products.visibleOn){
+                itemsToShowOnMobile = rexConsoleConfigs.products.visibleOn.mobile;
+            }else{
+                itemsToShowOnMobile = rexConsoleConfigs.products.visible;
+            }
             itemsToShow = itemsToShowOnMobile ? itemsToShowOnMobile : 2;
         }
-        if(window.unbxdDeviceType === "mobile-browser" || options.unbxdDeviceType === "mobile-browser"){
+        else if (device === MOBILE || browserSize === SMALL){
```
Comment (skumar-lohika) [inline src/index.js:571]: Create a common function for below statement and use it in both if and else if condition

Comment (deepunbxd005) [reply src/index.js:571]: i did not creat a seperate function for this as this is already inside an enclosed space and creating on one more would be an un necessary hassle

File: src/index.js (hunk 1 above)
Comment (deepunbxd005) [reply src/index.js:562]: not required,

**Pull request merged**